config/install/node.type.species.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
+name: Species
+type: species
+description: 'A list of Species Resources, using data from The Star Wars API(SWAPI)'
```

Comment:
Add a period at the end of the description to match others.

Reply:
Done and ready for review.